tests/envlint.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+import conda_build.metadata
+
+variants = { 'python' : ['3.6','3.7'], 'build_type' : ['cpu', 'cuda'] }
+
+def make_parser():
+    ''' Parser input arguments '''
+    parser = argparse.ArgumentParser(
+        description = 'Lint Environment Files',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument(
+        'env_files',
+        nargs='+',
+        type=str,
+        help="""Files to Lint.""")
+
+    return parser
+
+def main(arg_strings=None):
+    parser = make_parser()
+    args = parser.parse_args(arg_strings)
+
+    retval = 0
+    for env_file in args.env_files:
+        try:
+            meta_obj = conda_build.metadata.MetaData(env_file, variant=variants)
+            if not ("packages" in meta_obj.meta.keys() or "imported_envs" in meta_obj.meta.keys()):
+                raise Exception("Content Error!", "An environment file needs to specify packages or import another environment file.")
```

Comment:
@bnemanich 
Does it make sense to have this requirement?

Reply:
Are there any others that I should add?

Reply:
I think this one makes sense. Another possible one is to check for any unknown keys. Those probably won't cause an error, but it could catch things like `packages` being spelt `pckages`.

Reply:
Ooooh, that's a good one.
I'll add that.
Are the only possibilities:
```
imported_envs
channels
packages
```

Reply:
Also, we might want this to be part of the actual build tool as well, to check env files before they are loaded. Maybe the heart of this could be moved to the build directory, and this file would just import the function.

Reply:
That's a good idea.
I'll make that change.

Reply:
@bnemanich 
I made this change. Let me know what you think.